doc/classreference.rst

```diff
@@ -61,7 +61,6 @@ Selector Classes
     ParallelDirSelector
     DirectionSelector
     DirectionNthSelector
-    RadiusNthSelector
```

Comment:
Why was this removed? It was added recently in #504 wasn't it?

Reply:
Some kind of mistake, solved